docs/flashbots-protect/nonce-management.mdx

```diff
@@ -0,0 +1,85 @@
+---
+title: Nonce Management
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Normally, a wallet can call `eth_getTransactionCount` to get the next nonce to use for a transaction.
+However, since transactions sent to Flashbots Protect are potentially sensitive, even exposing the incremented nonce can leak information about the user's activity.
+
+As such, Flashbots Protect requires that all requests to `eth_getTransactionCount` be signed by the user when querying the `"pending"` nonce.
```

Comment:
I might specify that if the request is signed, the response will include pending private transactions in the tx count. If the request is unsigned, the response will only include pending public transactions. (Falls back to the default behavior).

Right now it sounds like you can't even get a response if you don't sign the request which isn't 100% true.

Reply:
Yup good call, will update!

Reply:
Updated!  I'll re-read it again for clarity tomorrow morning then merge.